### DISCUSSION TITLE
Ability to split the ingredients list into sections

### DISCUSSION THREAD

**Discussion by Dalarialus** [mealie-recipes/mealie#3613] [Feature Request]
Please disregard this feature request - I need to read the documentation better next time.

**Comment by Kuchenpirat** [comment]
Hey, this has been a feature for quite some time now. 
If you edit a reicpe use the 3-dot menue besides any ingredient and click "toggle section" which will allow you to set a title for that section. 
The same works for steps as well by the way.

Hope this answers your question and if you have any idea how to communicate this better feel free to leave feedback

**Reply by Dalarialus** [reply]
Sorry! I was literally editing my post while you were typing your reply - this appears to have been a clear-cut RTFM case :)

**Reply by Kuchenpirat** [reply]
don't worry, happens to the best, i'll go ahead and close this 👍
